.joss/paper.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'NiBetaSeries: task related correlations in fMRI'
+tags:
+  - python
+  - neuroscience
+  - neuroimaging
+  - BIDS
+  - fMRI
+authors:
+  - name: James D. Kent
+    orcid: 0000-0002-4892-2659
+    affiliation: 1
+affiliations:
+ - name: Neuroscience Program, University of Iowa
+   index: 1
+date: 30 January 2019
+bibliography: paper.bib
+---
+
+# Introduction
+
+You may not be aware of it, but your brain is orchestrating a complex ballet of activity while reading this sentence.
+Whether it is following a dot, reading a sentence, interpreting what response is required when presented a red square; the brain is evaluating input and sending motoric output to perform optimally/efficiently.
+We can measure and analyze this flurry of activity using functional Magnetic Resonance Imaging (fMRI).
+Traditional fMRI analysis emphasizes what regions are "activated/deactivated" during a task, but does not provide information on which regions are acting in synchrony.
+Knowing what regions are synchronous during a task gives insights on the potential organization of the brain.
+Additionally, the information could open new inquiries about how different tasks show different synchronicity and whether synchrony between brain regions correlates with performance on a particular task.
```

Comment:
this seems like the same point as above, suggest keeping the first one since it's more direct